website/docs/getting-started/installation.mdx

```diff
@@ -0,0 +1,135 @@
+- **'taqueria-plugin-smartpy'**
+    - [Python](https://www.python.org/) availble for download [here](https://www.python.org/downloads/)
+    - [SmartPY](https://smartpy.io/) which is installed as detailed in [this guide](https://smartpy.io/docs/cli/#direct-installation)
+- **'taqueria-plugin-flextesa'**
+    - Requires a recent version of [Docker](https://www.docker.com/)
+
+### VS Code Extension
+
+The VS Code Extension requires a recent version of Microsoft Visual Studio Code which can be installed [here](https://code.visualstudio.com/) 
+
+## Installing the Taqueria CLI
+
+### Overview
+
+Installing the Taqueria CLI involves the following steps:
+
+1. Download the correct build of Taqueria for your operating system to an installation directory of your choice
+2. Make the Taqueria binary ('taq') executable
+3. Update your shell's $PATH to include the Taqueria installation directory
+4. Configure your OS security settings to allow the Taqueria CLI/SDK (taq) to run
+
+:::note
+The Taqueria executable is not currently code signed for Mac or Windows operating systems and at this time you will recieve warnings about 'insert actual error message' from your OS.
+
+You must explicitly allow the taq executable to run using the reccomended solutions detailed in the OS specific installation instructions
+:::
+
+Once these steps are complete, you will be able to run Taqueria from your terminal using the 'taq' command
+
+Step by step instructions for installing the Taqueria CLI on a Windows, MacOS, or Linux machine are detailed in the following section. If you are comfortable with downloading the unsigned Taqueria binary and configuring it to run in your terminal on your own, you can find the correct build for your local machine [here](https://github.com/ecadlabs/taqueria/releases), and then skip to the [Installing Taqueria Plugins Section](#installing-taqueria-pluginsz) 
+
+### OS Specific Installation Instructions
+
+To make installation as fast as possible, you can follow the steps detalled below for your particular operating system. Note that these steps are simply one method for installation. You can adjust the particular directories and shell configuration if you feel comfortable doing so
+
+The following steps assume the following terminals on each operating system:
+- MacOS: zsh
+- Windows: console???
+- Linux: bash
+
+
+<Tabs groupId="operating-systems">
+<TabItem value="mac" label="MacOS">
+  
+1. Create a directory to install the Taqueria binary and change into that directory
+
+2. Download the latest MacOS build of Taqueria from the [releases page](https://github.com/ecadlabs/taqueria/releases/) and name the binary file 'taq'
+```shell
+curl https://github.com/ecadlabs/taqueria/releases/download/untagged-d7ba60ff3f2af73f46c7/taqueria.v0.0.1-test.18-macos --output taq
+```
+
+3. Make the binary file executable
+```shell
+chmod -x taq
+```
+
+4. Add the installation directory to your $PATH
+
+5. Disable MacOS Gatekeeper to allow Taqueria to run
```

Comment:
This step worked nicely for me in testing. Lets add a Caution note here;

> Currently the `taq` binary is not signed for Mac OS, so in-order to run the command, users must disable the MacOS Gatekeeper. (This is not ideal). This is a temporary workaround until we can sign the binaries (see #129). If you want to re-enable the MacOS Gatekeeper later on, run the command `spctl HOW DO YOU RENEABLE THIS??`

Reply:
Will do. Great wording here👌